WordPress/Classes/ViewRelated/Jetpack/Install/JetpackRemoteInstallViewController.swift

```diff
@@ -58,12 +55,10 @@ private extension JetpackRemoteInstallViewController {
 
         jetpackView.delegate = self
         add(jetpackView)
+        jetpackView.view.frame = view.bounds
```

Comment:
Why wasn't this necessary previously?

Reply:
Because of the previous constraints I removed with the segmented. Without the view is taking the size of the content. It's part of the next PR I'm still working on it 😊